### ISSUE TITLE
No api to share a group with another group

### ISSUE THREAD

**Issue by fdsch** [gitlab4j/gitlab4j-api#587]
It seems like the api to [share a group with another group](https://docs.gitlab.com/ee/api/groups.html#share-groups-with-groups): `POST /groups/:id/share` is not implemented.

**Comment by gmessner** [comment]
@fdsch Support for sharing/unsharing a group with another group is now available in release 4.15.2. Please verify and close this issue when you have time.

Here are the methods added to ```GroupApi```:
```
public Group shareGroup(Object groupIdOrPath, Integer shareWithGroupId, AccessLevel groupAccess, Date expiresAt)
public void unshareGroup(Object groupIdOrPath, Integer sharedWithGroupId)
```

**Comment by gmessner** [comment]
@fdsch Since this functionality was released a week ago I am closing this issue.  If you have any problems with it please feel free to re-open.
